install/build_from_source.md

```diff
@@ -30,6 +30,16 @@ The following are the basic instructions for UNIX-like systems. We will use the
 # substitute `v6-22-00-patches` with the patches branch of the latest release
 $ git clone --branch v6-22-00-patches https://github.com/root-project/root.git root_src
 ```
+
+If you are interested in the latest stable (as opposed to a specific) version, you can get a copy as follows:
+```bash
+$ git clone --branch stable https://github.com/root-project/root.git root_src
+```
+This branch is expected to be updated regularly on each release. Therefore, users who cloned this branch can upgrade to the latest release in a straightforward way:
+```bash
+$ git pull
+```
```

Comment:
I guess we can completely get rid of the command that checks out `v6-22-00-patches` in favor of this one, which works independently of what the latest stable is (yes, `stable` does not include the latest patches, but that's a price I'm glad to pay so we have an `install` documentation that's independent from the ROOT versions)

Reply:
Hmmm...  @Axel-Naumann, +1 or not?

Reply:
@Axel-Naumann also suggested a blog post to gain more visibility of this new branch.  However, I would make it in a different PR once the `stable` branch has been created, which is expected for the next release.

Reply:
:+1: for doing a blog post, after the `stable` branch is there. But I think we should start the `stable` branch for v6.24/00, right?